projects/projects.json

```
@@ -3,7 +3,7 @@
         {
             "name": "falco",
             "organization": "falcosecurity",
-            "benchmark_path": "",
+            "benchmark_path": "./.github/workflows/benchmark-workflows/falco.yml",
```

```suggestion
            "benchmark_path": "cncf-tags/green-reviews-tooling/.github/workflows/benchmark-workflows/falco.yml",
```

`dynamic-uses` wants the workflow name to be fully qualified.